CHANGELOG.md

```diff
@@ -19,13 +19,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [`0.7.0-preview`] - 2019-10-11
 
 ### New Known Issues:
-- MSVC v14.23 removes `typeinfo.h` and replaces it with `typeinfo`. This change causes errors when building the Unreal Engine. This issue affects Visual Studio 2019 users. Until [this proposed fix](https://github.com/EpicGames/UnrealEngine/pull/6226) is accepted by Epic Games, you can work around the issue by:
```

Comment:
removed the link to the Epic Games PR as that change has merged, and I do not want to give users the incorrect idea that because of this, out engine fork now contains that fix.